packages/mint-css/base/utility.css

```diff
@@ -0,0 +1,185 @@
+.hide {
+  display: none;
+}
+
+.absolute-center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.flex {
+  display: flex;
+}
+
+.valign-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.halign-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.vspace-between {
+  justify-content: space-between;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.right-align {
+  text-align: right;
+}
+
+.left-align {
+  text-align: left;
+}
+
+.center-align {
+  text-align: center;
+}
+
+.responsive-img {
+  max-width: 100%;
+  height: auto;
+}
+
+.onMount-appear {
+  opacity: 0.01;
+}
+
+.onMount-appear.onMount-appear-active {
+  opacity: 1;
+  transition: opacity 0.5s ease-in;
+}
+
+.truncate {
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.web-align {
+  max-width: 1110px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.dropdown {
+  display: inline-block;
+}
+
+.dropdown__content {
+  display: none;
+  position: absolute;
+}
+
+.dropdown--active .dropdown__content {
+  display: block;
+}
+
+.pos-rel {
+  position: relative;
+}
+
+.pos-abs {
+  position: absolute;
+}
+
+.circle {
+  border-radius: 50%;
+}
+
+.fullWidth {
+  width: 100%;
+}
+
+.page-padding {
+  padding: 0 16px;
+}
+
+
+.cur-po {
+  cursor: pointer;
+}
+
+.cur-no {
+  cursor: not-allowed;
+}
+
+.flo-r {
+  float: right;
+}
+
+.flo-l {
+  float: left;
+}
+
+.width100 {
+  width: 100%;
+}
+
+.card {
+  border-radius: 5px;
+  box-shadow: 0 1px 5px 0 var(--boxShadow);
+  border: 1px solid var(--border);
+}
+
+.clickable-text {
```

Comment:
check for tokens

Reply:
This PR has only package changes, we have another PR which has migration changes, which we will be merging today only after this PR